client/src/js/components/bhStrengthMeter.js

```diff
@@ -0,0 +1,21 @@
+angular.module('bhima.components')
+.component('bhStrengthMeter', {
+  template :
+    "<div class='strength-meter'>"+
+    " <div class='strength-meter-fill' data-strength='{{$ctrl.counter()}}'></div>"+
+    "</div>",
+  bindings : {
+    password : '=',
```

Comment:
Is there any reason for this to be a [two-way binding](https://docs.angularjs.org/guide/component#component-based-application-architecture)?  Can it be a one-way binding (`<`)?